src/channel.ts

```diff
@@ -51,19 +51,22 @@ export const OPACITY = Channel.OPACITY;
 export const CHANNELS = [X, Y, X2, Y2, ROW, COLUMN, SIZE, SHAPE, COLOR, ORDER, OPACITY, TEXT, DETAIL];
 
 // CHANNELS without COLUMN, ROW
-export const UNIT_CHANNELS = [X, Y, X2, Y2, SIZE, SHAPE, COLOR, ORDER, OPACITY, TEXT, DETAIL];
+export const UNIT_CHANNELS = without(CHANNELS, [ROW, COLUMN] as Channel[]);
```

Comment:
@domoritz  actually prefer the old one. 

This makes type of UNIT_CHANNELS become any[] instead of  ('x'|'y'|...)[]